00:00:00.004727 main     VirtualBox COM Server 6.1.18 r142142 win.amd64 (Jan  7 2021 15:32:19) release log
00:00:00.004732 main     Log opened 2021-02-22T01:27:04.315556200Z
00:00:00.004733 main     Build Type: release
00:00:00.004741 main     OS Product: Windows 10
00:00:00.004742 main     OS Release: 10.0.19041
00:00:00.004743 main     OS Service Pack: 
00:00:00.114594 main     DMI Product Name: Inspiron 5505
00:00:00.133433 main     DMI Product Version: 1.4.0
00:00:00.133446 main     Firmware type: UEFI
00:00:00.133760 main     Secure Boot: VERR_PRIVILEGE_NOT_HELD
00:00:00.133784 main     Host RAM: 7545MB (7.3GB) total, 3311MB (3.2GB) available
00:00:00.133789 main     Executable: C:\VBoxSVC.exe
00:00:00.133790 main     Process ID: 19736
00:00:00.133791 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.239256          VirtualBox: object creation starts
00:00:00.239654          Home directory: 'C:\Users\bri23\.VirtualBox'
00:00:00.239837          Installed Drivers:
00:00:00.246116            C:\Windows\system32\DRIVERS\VBoxDrv.sys (Version: 6.1.18.42142)
00:00:00.246665            C:\Windows\system32\DRIVERS\VBoxUSBMon.sys (Version: 6.1.18.42142)
00:00:00.247246            C:\Windows\system32\DRIVERS\VBoxNetAdp6.sys (Version: 6.1.18.42142)
00:00:00.247823            C:\Windows\system32\DRIVERS\VBoxNetLwf.sys (Version: 6.1.18.42142)
00:00:00.248339          Loading settings file "C:\Users\bri23\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.250798          usbLibGetDevices: Starting USB device enumeration
00:00:00.251153          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&2f6e09c7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 2
00:00:00.252639          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#5&2f6e09c7&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 4
00:00:00.254223          usbLibGetDevices: Found 3 USB devices, 0 captured
00:00:00.254291          USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0x565a)
00:00:00.254334          USB: Unknown USB device detected (idVendor: 0x27c6, idProduct: 0x538d)
00:00:00.254349          USB: Unknown USB device detected (idVendor: 0x8087, idProduct: 0x0029)
00:00:00.403076          HostDnsMonitor: initializing
00:00:00.413662          HostDnsMonitor: updating information
00:00:00.413730          HostDnsMonitor: old information
00:00:00.413731            no server entries
00:00:00.413732            no domain set
00:00:00.413732            no search string entries
00:00:00.413733          HostDnsMonitor: new information
00:00:00.413734            server 1: 1.1.1.1
00:00:00.413735            server 2: 8.8.8.8
00:00:00.413736            no domain set
00:00:00.413736            no search string entries
00:00:00.488816          VD: VDInit finished with VINF_SUCCESS
00:00:00.490461          Loading settings file "C:\Users\bri23\VirtualBox VMs\Ubuntu 18.04.5\Ubuntu 18.04.5.vbox" with version "1.16-windows"
00:00:00.491888          VirtualBox: object created
00:00:10.093508          Launched VM: 67293568 pid: 15008 (0x3aa0) frontend: GUI/Qt name: Ubuntu 18.04.5
00:00:10.875437 Watcher  ERROR [COM]: aRC=E_FAIL (0x80004005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={MachineWrap} aText={The virtual machine 'Ubuntu 18.04.5' has terminated unexpectedly during startup with exit code -1073741424 (0xc0000190).  More details may be available in 'C:\Users\bri23\VirtualBox VMs\Ubuntu 18.04.5\Logs\VBoxHardening.log'}, preserve=false aResultDetail=0
00:01:17.607588          Saving settings file "C:\Users\bri23\VirtualBox VMs\Ubuntu 18.04.5\Ubuntu 18.04.5.vbox" with version "1.16-windows"
00:01:17.650673          Saving settings file "C:\Users\bri23\VirtualBox VMs\Ubuntu 18.04.5\Ubuntu 18.04.5.vbox" with version "1.16-windows"
00:01:18.979997          Launched VM: 67546240 pid: 13924 (0x3664) frontend: GUI/Qt name: Ubuntu 18.04.5
00:01:19.786072 Watcher  ERROR [COM]: aRC=E_FAIL (0x80004005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={MachineWrap} aText={The virtual machine 'Ubuntu 18.04.5' has terminated unexpectedly during startup with exit code -1073741424 (0xc0000190).  More details may be available in 'C:\Users\bri23\VirtualBox VMs\Ubuntu 18.04.5\Logs\VBoxHardening.log'}, preserve=false aResultDetail=0
00:05:36.208486          Launched VM: 67547584 pid: 21284 (0x5324) frontend: GUI/Qt name: Ubuntu 18.04.5
00:05:37.045650 Watcher  ERROR [COM]: aRC=E_FAIL (0x80004005) aIID={85632c68-b5bb-4316-a900-5eb28d3413df} aComponent={MachineWrap} aText={The virtual machine 'Ubuntu 18.04.5' has terminated unexpectedly during startup with exit code -1073741424 (0xc0000190).  More details may be available in 'C:\Users\bri23\VirtualBox VMs\Ubuntu 18.04.5\Logs\VBoxHardening.log'}, preserve=false aResultDetail=0
00:05:45.633962          Saving settings file "C:\Users\bri23\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:05:45.655119          Saving settings file "C:\Users\bri23\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:05:51.172421 main     VirtualBox: object deletion starts
00:05:51.172554 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'C:\Users\bri23\VirtualBox VMs\Ubuntu 18.04.5\Ubuntu 18.04.vdi' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
00:05:51.174453 main     HostDnsMonitor: shutting down ...
00:05:51.174563 main     HostDnsMonitor: shut down
00:05:51.184436 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={d0a0163f-e254-4e5b-a1f2-011cf991c38d} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
00:05:51.184762 main     VirtualBox: object deleted
